.github/workflows/cloud_build_failure_reporter.yml

```diff
@@ -133,8 +135,8 @@ jobs:
                       // error case
                       else {
                         createOrCommentIssue(
-                          'Failing periodic',
-                          `[Failing periodic](${periodicCheck.html_url}) at ${commit.html_url}.`
+                          '[Cloud Build Failure Reporter] test failing for ${check.name}',
+                          `Cloud Build Failure Reporter found test failure for ${check.name} (${periodicCheck.html_url}) at ${commit.html_url}. Please fix the error and then close the issue after the ${check.name} test passes.`
```

Comment:
Please use markdown to format the links properly.

Reply:
updated